app/_components/common/ToggleSwitch.tsx

```diff
@@ -0,0 +1,134 @@
+import palette from '@/_styles/palette';
```

Comment:
```suggestion
import palette from '@styles/palette';
```
`tsconfig.json`에 정의된 path로 변경하면 좋을 것 같아요!